Formula/i/iota.rb

```diff
@@ -0,0 +1,37 @@
+class Iota < Formula
+    desc "Bringing the real world to Web3 with a scalable, decentralized and programmable DLT infrastructure"
+    homepage "https://www.iota.org"
+    url "https://github.com/iotaledger/iota/archive/refs/tags/v0.8.1-rc.tar.gz"
```

Comment:
We don't want to ship release candidates, from the requirements page:

> be stable (e.g. not declared “unstable” or “beta” by upstream)

https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff:~:text=be%20stable%20(e.g.%20not%20declared%20%E2%80%9Cunstable%E2%80%9D%20or%20%E2%80%9Cbeta%E2%80%9D%20by%20upstream)

Reply:
Hi Sean, in this case RC is about the deployment and not the code readiness. Our deployment goes from Devnet (alpha/beta, unstable) to Testnet (rc, stable) to Mainnet (stable). RC is a finished and stable release but just not yet deployed on our Mainnet.
This is very similar to the `sui` formula but they have opted for a `testnet-` suffix.

Reply:
That one shouldn't ship testing level software either, we might want to disable that one.